.github/workflows/zscaler-iac-scan.yml

```
@@ -33,6 +33,7 @@ jobs:
       contents: read # for actions/checkout to fetch code
       security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
     runs-on: ubuntu-latest
+    if: ${{ secrets.ZSCANNER_CLIENT_ID != '' && secrets.ZSCANNER_CLIENT_SECRET != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)